pkg/tsdb/azuremonitor/applicationinsights-datasource.go

```diff
@@ -144,10 +133,9 @@ func (e *ApplicationInsightsDatasource) buildQueries(queries []plugins.DataSubQu
 	return applicationInsightsQueries, nil
 }
 
-// nolint:staticcheck // plugins.DataQueryResult deprecated
 func (e *ApplicationInsightsDatasource) executeQuery(ctx context.Context, query *ApplicationInsightsQuery) (
-	plugins.DataQueryResult, error) {
-	queryResult := plugins.DataQueryResult{Meta: simplejson.New(), RefID: query.RefID}
+	backend.DataResponse, error) {
+	queryResult := backend.DataResponse{}
```

Comment:
Maybe we can rename this variable to dataResponse or something so that it reflects the new type name?